spec/rails_helper.rb

```diff
@@ -62,4 +63,12 @@
 
   # serializer_spec_helper
   config.include SerializerSpecHelper, type: :serializer
+
+  # clean up carrierwave uploads
+  # https://github.com/carrierwaveuploader/carrierwave/wiki/How-to:-Cleanup-after-your-Rspec-tests
+  config.after(:all) do
+    if Rails.env.test?
+      FileUtils.rm_rf(Dir["#{Rails.root}/spec/support/uploads"])
+    end
+  end
```

Comment:
Not sure I understand, this is a global teardown that cleans up any files that were placed in the `spec/support/uploads` folder (the location used by CarrierWave in tests)